services/viva-ms/lambdas/createVivaCase.js

```diff
@@ -123,6 +123,7 @@ async function putRecurringVivaCase(vivaPerson) {
 
   const initialFormAttributes = {
     answers: [],
+    encryption: { type: 'decrypted' },
     currentPosition: {
       currentMainStep: 1,
       currentMainStepIndex: 0,
```

Comment:
Maybe this could be extracted to a lib function that has logic for some general initialisation that goes for all future services? The function could then take in a callbackfunction or something to make it possible for the service to add or remove specific logic/options

Not required at this point in time :)